core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -81,7 +81,7 @@
   /**
    * min blocklet size
    */
-  public static final int BLOCKLET_SIZE_MIN_VAL = 50;
+  public static final int BLOCKLET_SIZE_MIN_VAL = 2000;
```

Comment:
Number of rows in blocklet is exposed for V1 and V2 format so if user configure and any value we should validate whether is value is valid because of that min and max value is added